CMakePresets.json

```diff
@@ -7,18 +7,18 @@
     },
     "configurePresets": [
         {
-            "name": "config-common",
+            "name": "common",
             "description": "General settings that apply to all configurations",
             "hidden": true,
             "generator": "Ninja Multi-Config",
-            "binaryDir": "${sourceDir}/out/build/${presetName}",
-            "installDir": "${sourceDir}/out/install/${presetName}"
+            "binaryDir": "${sourceDir}/build",
+            "installDir": "${sourceDir}/out/stage"
         },
         {
-            "name": "config-windows-common",
+            "name": "windows-common",
```

Comment:
for the hidden presets that are used to compose the non-hidden presets (using the "inherits") I would prefer to keep the prefix type to prevent any misuse. Those hidden names are not visible to the end user anyway.

Reply:
`configure, build,` and `test `preets are independent **namespaces**!

you can't inherit a `build preset `from a `configure preset` **AFAIK**!

Reply:
thats' a good point. I changed them all in the most recent version!